src/managers/GuildMemberManager.js

```diff
@@ -263,8 +263,11 @@ class GuildMemberManager extends CachedManager {
     }
     _data.roles &&= _data.roles.map(role => (role instanceof Role ? role.id : role));
 
-    _data.communication_disabled_until =
-      _data.communicationDisabledUntil && new Date(_data.communicationDisabledUntil).toISOString();
+    if (_data.communicationDisabledUntil) {
+      _data.communication_disabled_until = new Date(_data.communicationDisabledUntil).toISOString();
+    } else if (_data.communicationDisabledUntil === null) {
+      _data.communication_disabled_until = null;
+    }
```

Comment:
Only need to wrap the existing code around a `typeof` check:

```suggestion
    if (typeof _data.communicationDisabledUntil !== 'undefined') {
      _data.communication_disabled_until =
        _data.communicationDisabledUntil && new Date(_data.communicationDisabledUntil).toISOString();
    }
```